packages/synthetix-main/contracts/mocks/token/SNXTokenMock.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@synthetixio/core-contracts/contracts/proxy/UUPSImplementation.sol";
+import "@synthetixio/core-contracts/contracts/ownership/Ownable.sol";
+import "@synthetixio/core-contracts/contracts/token/ERC20.sol";
+
+contract SNXTokenMock is Ownable, UUPSImplementation, ERC20 {
```

Comment:
See comment below. Needed for the test I added.

Reply:
Shouldn't this extend SNXToken instead?